### DISCUSSION TITLE
Load more posts by scrolling to bottom?

### DISCUSSION THREAD

**Discussion by candyho-learning** [yogeshwaran01/instagramy#19] [Q&A] [answered]
Hi there,

A big thank you to the creator of Instagramy! It's been helpful.

I've been trying to scrape more posts per user (now the limit is 12, I assume because of the infinite scroll on Instagram pages)? Does anyone know a way to modify the code somehow so the html from the request is returned *after* we scroll to the bottom of the page?

Thanks in advance!

**Comment by yogeshwaran01** [comment, marked as answer]
@candyho-learning Sorry for the late reply

It is not possible by instagramy. it scrapes only the static content of the Instagram page. It is possible with [selenium](https://selenium-python.readthedocs.io) or try [Instapy](https://pypi.org/project/instapy/)
